solutions/io.lionweb.mps.json.test/models/io.lionweb.mps.json.test.json2lioncore@tests.mps

```diff
@@ -160,7 +160,7 @@
       <concept id="2656571587264858085" name="io.lionweb.mps.m3.structure.Feature" flags="ng" index="2RzOSH">
         <property id="2656571587264858528" name="optional" index="2RzO1C" />
       </concept>
-      <concept id="2656571587264863094" name="io.lionweb.mps.m3.structure.Interface" flags="ng" index="2RzPaY">
+      <concept id="2656571587264863094" name="io.lionweb.mps.m3.structure.ConceptInterface" flags="ng" index="2RzPaY">
```

Comment:
This seems to undo an earlier renaming.

Reply:
Good catch. I am not sure why it happened but I opened the file, and then MPS re-updated the name in .mps file... so now it should be back to the proper value